epaint/src/tessellator.rs

```diff
@@ -286,6 +286,9 @@ pub struct TessellationOptions {
 
     /// If true, no clipping will be done.
     pub debug_ignore_clip_rects: bool,
+
+    //to add this since it will cause a lot of breaking changes.
+    pub bezier_flattern_tolerence: f32,
```

Comment:
This needs a docstring explaining it. What is the tolerance a metric for? Max distance error?

Reply:
Added explanation to the `Tolerance` and it's the max distance between the curve and approximated line. 
I added a new option `epsilon` which is used when comparing floats.